### ISSUE TITLE
Adam optimizer and Batch norm parameters

### ISSUE THREAD

**Issue by chtouroumahdi** [AlexeyAB/darknet#4000]
Hi,
i'm using tiny yolo v2 and im traying to use adam optimizer during training , so i added the following lines in cfg
![image](https://user-images.githubusercontent.com/33591581/65779761-abef6a80-e148-11e9-9597-0baf92675563.png)

but by default there is also this param in cfg :
![image](https://user-images.githubusercontent.com/33591581/65779868-db05dc00-e148-11e9-81f6-60ee2ceb31df.png)
Should i delete them because i think there is no need for momentum when using adam ...

the second question is i want to set the parameters of batch normalization layers to 
eps = 0.001
momentum = 0.99

so how can i do it in cfg file , and is there knowing that i need always to use eps = 0.0000001 for adam optimizer and eps =0.001 for batch norm ...

i also know that adam use decay = 0 so should i set decay in cfg from 0.005 to 0?

**Comment by spaul13** [comment]
@chtouroumahdi I also want to assign ADAM optimizer in .cfg file and set the parameters of the batchnorm layers like yours. Did u find out the solution of your question?
